Library/Formula/rpm.rb

```diff
@@ -52,18 +56,20 @@ def install
         --with-neon=external
         --with-uuid=external
         --with-pcre=external
-        --with-lua=internal
-        --with-syck=internal
+        --with-lua=external
+        --with-syck=external
```

Comment:
Maintaining statically linked / internal libraries is a pain, but we can do it if you really want to. :-)
However, some libraries like berkeley-db are really really painful to handle, so leaving them as-is.